common/focuses/wc_magic_lifestyle_focuses.txt

```diff
@@ -0,0 +1,927 @@
+﻿light_magic_focus_1 = {
+	lifestyle = light_magic_lifestyle
+
+	is_valid = {
+		MAGIC_lifestyle_is_valid_trigger = { MAGIC = light }
```

Comment:
Explain why here it's `MAGIC_lifestyle_is_valid_trigger` and in the perks it's `can_openly_learn_elemental_air_magic_trigger = no`, etc.

Reply:
So the AI can pick the focus but doesn't pick perks?

Reply:
Changed trigger in perks to `MAGIC_lifestyle_is_valid_trigger`

Reply:
Thanks for noticing